cmd/send.go

```
@@ -2,19 +2,19 @@ package cmd
 
 import (
 	"github.com/claudiodangelis/qrcp/config"
-	"github.com/claudiodangelis/qrcp/logger"
 	"github.com/claudiodangelis/qrcp/payload"
 	"github.com/claudiodangelis/qrcp/qr"
+	log "github.com/sirupsen/logrus"
 
 	"github.com/claudiodangelis/qrcp/server"
 	"github.com/spf13/cobra"
 )
 
 func sendCmdFunc(command *cobra.Command, args []string) error {
-	log := logger.New(quietFlag)
+	// log := logger.New(quietFlag)
 	payload, err := payload.FromArgs(args, zipFlag)
 	if err != nil {
-		return err
```

I would prefer errors being returned rather than having the execution aborted.

---

Isnt that what `log.Fatal(err)` does? to quote from the documentation, 
> Fatal logs a message at level Fatal on the standard logger then the process will exit with status set to 1.

---

I'm saying the opposite of that: don't let the function exit the process, but rather return the error and let the caller function handle it. Does it make sense?

---

Ah, got it. Makes sense